widgets/side_widget.py

```diff
@@ -58,6 +59,12 @@ def __init__(self, *args, **kwargs):
         self.verticalLayout.addWidget(self.button_remove_object)
         self.verticalLayout.addWidget(self.button_ground_object)
         #self.verticalLayout.addWidget(self.button_move_object)
+
+        self.more_buttons = MoreButtons(parent)
+        self.more_buttons.add_buttons(None)
+        self.more_buttons.setMinimumSize(self.button_add_object.width(), 30)
```

Comment:
Question: Why was this fixed size needed?

I'm not sure we should set the size with static units. Depending on the user display, 30 pixels can be either too small or too large. Best to let Qt do the work. If the user shrinks the pane too much, that's on them!